rov/thrusters/__init__.py

```diff
@@ -0,0 +1,16 @@
+from thrust_mapper import ThrustMapper
+
+# Attemps to initialize the hardware thrustesr
+# If it fails on init, then use the mock thrusters
+
+
+def Thrusters():
```

Comment:
Did also move all the thruster related things to `rov/thrusters` so this was possible